src/muse/investments.py

```diff
@@ -310,8 +310,6 @@ def scipy_match_demand(
 
     from muse.constraints import ScipyAdapter
 
-    df_technologies = technologies.to_dataframe()
```

Comment:
So... is this not needed anymore?

Reply:
Nope. Previously it was converting the xarray to a dataframe and then getting the list of technologies from the dataframe, but you can get this directly from the xarray